src/utility/algod.utility.ts

```diff
@@ -11,9 +11,9 @@ export const fundAmount = async (
 	from: string,
 	to: string,
 	amount: number,
-	webMode: WebMode
+	webMode: any // eslint-disable-line
```

Comment:
Can't we make it type `WebModeTypes`?

Reply:
No we can’t because after connecting the wallet new type is assigned to my algo connect and wallet connect